src/authenticated/member/MemberProfileEditPage.js

```diff
@@ -44,18 +45,41 @@ function prepUserForUpdate(updatedUser) {
   };
 }
 
-export default compose(
-  withLoggedInUser,
-  graphql(updateUserQuery, {
-    props: ({ mutate }) => ({
-      onEdit: updatedUser => {
-        return mutate({ variables: prepUserForUpdate(updatedUser) });
-      }
-    })
-  }),
-  withEditPage({
-    renameProps: {
-      formData: 'user'
-    }
-  })
-)(MemberProfile);
+class MemberProfileEditPage extends Component {
+  render() {
+    return (
+      <AuthenticationContext.Consumer>
+        {context => (
+          <Mutation
+            mutation={updateUserQuery}
+            onCompleted={({ updateUser }) => {
+              context.setCurrentUserProfile(updateUser);
```

Comment:
How does this get on here? I don't see `setCurrentUserProfile` in the context provider value

Reply:
The `Query` component could be part of the `AuthenticationProvider` component. Not sure if that would be a bad idea or not...

Reply:
Oh, nevermind on the question of how it gets there. I see it now